superset-frontend/src/views/CRUD/chart/ChartList.tsx

```diff
@@ -45,6 +45,7 @@ import withToasts from 'src/messageToasts/enhancers/withToasts';
 import PropertiesModal from 'src/explore/components/PropertiesModal';
 import Chart from 'src/types/Chart';
 import TooltipWrapper from 'src/components/TooltipWrapper';
+import { destroyBulk as bulkDeleteCharts } from 'src/api/chart';
```

Comment:
This is another case where I think we might as well name the export in this explicit manner, then it'll be easier to search/track its use cases in the codebase.